core/src/test/java/org/apache/iceberg/actions/TestBinPackStrategy.java

```diff
@@ -150,6 +167,23 @@ public void testGroupingMinInputFilesValid() {
         ImmutableList.of(testFiles), grouped);
   }
 
+  @Test
+  public void testGroupingWithDeletes() {
+    RewriteStrategy strategy = defaultBinPack().options(ImmutableMap.of(
+            BinPackStrategy.MIN_INPUT_FILES, Integer.toString(5),
+            BinPackStrategy.MAX_FILE_SIZE_BYTES, Long.toString(550 * MB),
+            BinPackStrategy.MIN_FILE_SIZE_BYTES, Long.toString(490 * MB),
+            BinPackStrategy.MIN_DELETES_PER_FILE, Integer.toString(2)
```

Comment:
@kbendick, can you check our checkstyle config? It looks like lines that have incorrect indentation are getting through.